lib/dolly/document_type.rb

```diff
@@ -48,6 +48,10 @@ def self.included(base)
     end
 
     module ClassMethods
+      def absolute_id(id)
+        id.sub(%r{^[^/]+/}, '')
```

Comment:
this works with https://github.com/amco/dolly/commit/05eecec7d8c04af1ae81133e25f6ae27f288ae8c too?

Reply:
No, and I think this code is not related so I removed